national-resources/netherlands.md

```diff
@@ -0,0 +1,213 @@
+  - name: COVID RADAR app
+    description: Over a period of approximately 2 years (starting May 2020 until
+      February 2022), a total of 280.000 Dutch inhabitants filled in a short
+      questionnaire about Corona-related symptoms, behaviour, vaccination status
+      and test result.
+    how_to_access: download from link
+    instance_of: null
+    related_pages: null
+    registry: null
+    url: https://doi.org/10.17026%2Fdans-zcd-m9dh
+
+---
+<!---All the resources added above will appear on the table at the bottom of the page--->
+
+<!---Following information for the page text--->
+<!---If the information is already in another resource, please include the link instead of duplicating information--->
+<!---Please focus on resources that are relevant for the whole country for infectious diseases--->
+
+## Introduction 
+ 
+This page discusses infectious disease research in the Netherlands and provides information on relevant health authorities and initiatives. In addition, the page offers a collection of key considerations for researchers such as regulations for data handling and disease control. While the primary target audience is the scientific community in the Netherlands, you will also find (open-access) resources that could be useful for the wider community. 
+ 
+## Health authorities 
+ 
+<!--- A section to list and provide context to agencies/authorities/institutions which define public health measures and policies ---> 
+The Ministry of Health, Welfare and Sport ([VWS](https://www.government.nl/ministries/ministry-of-health-welfare-and-sport) in Dutch) and National Institute for Public Health and the Environment ([RIVM](https://www.rivm.nl/en)) work jointly to provide an effective and coordinated effort towards the prevention and control of infectious diseases. They also ensure that public health data is collected, stored, and shared in compliance with applicable laws and regulations. 
+ 
+The Ministry of VWS monitors public health issues to develop national guidelines and strategies for surveillance, prevention, and response activities. Together with local health authorities and healthcare providers, the ministry implements relevant policies and communicates these guidelines to the public. Furthermore, the ministry coordinated the national response to the COVID-19 pandemic, establishing national recommendations for testing, tracing, and isolation. 
+ 
+The RIVM advises health authorities on public health issues, including disease prevention, outbreak control, and preparedness for national health emergencies. Additionally, the RIVM works with the Ministry of VWS to develop vaccination programs, guide infection control measures, and support healthcare providers and related institutions. The [Centre for Infectious Disease Control](Centre for Infectious Disease Control | RIVM) within the RIVM supports infectious disease research, monitors infectious disease outbreaks, and analyses data related to these outbreaks. 
```

Comment:
```suggestion
The RIVM advises health authorities on public health issues, including disease prevention, outbreak control, and preparedness for national health emergencies. Additionally, the RIVM works with the Ministry of VWS to develop vaccination programs, guide infection control measures, and support healthcare providers and related institutions. The [Centre for Infectious Disease Control](https://www.rivm.nl/en/about-rivm/organisation/centre-for-infectious-disease-control) within the RIVM supports infectious disease research, monitors infectious disease outbreaks, and analyses data related to these outbreaks. 
```

I have no idea if this is the correct url, but the pipe character was causing some problems/ there was no real url given

Reply:
That must have gone wrong when copying the url. The url you added is correct.